cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/validator/AwsIDBrokerMappedRolePermissionValidator.java

```diff
@@ -128,6 +128,13 @@ public void validate(AmazonIdentityManagementClient iam, CloudS3View cloudFileSy
                         String.join(", ", roles.stream().map(Role::getArn).collect(Collectors.toCollection(TreeSet::new))),
                         String.join("\n", failedActions.stream().limit(MAX_SIZE).collect(Collectors.toSet())));
 
+                if (validationErrorMessage.contains(AbstractAwsSimulatePolicyValidator.DENIED_BY_ORGANIZATION_RULE)) {
+                    validationErrorMessage = String.join(validationErrorMessage,
```

Comment:
I assume you wanted to use `String.format` or concat.
Join will not work in this case. The original message will be lost.

Reply:
Yes, good catch, thank you.